web/src/stream_popover.js

```diff
@@ -89,11 +85,6 @@ export function all_messages_sidebar_menu_handle_keyboard(key) {
     popovers.popover_items_handle_keyboard(key, items);
 }
 
-export function starred_messages_sidebar_menu_handle_keyboard(key) {
```

Comment:
Deleted as it looks like unused. Also I am not sure where to put this logic at this moment.

Reply:
So the behavior that this block of code is related to is, if you click to open the popover, and hit the `Down` arrow key, there's basic keyboard navigation between the options (and `Enter` lets you select one). I don't like how duplicated the existing code was for this functionality in popovers, so it may be worth implementing in a slightly different way... but I'm not sure how you got the idea it was unused -- it's called from `hotkey.js`:
```
$ git grep starred_messages_sidebar_menu_handle_keyboard
web/src/hotkey.js:        stream_popover.starred_messages_sidebar_menu_handle_keyboard(event_name);
web/src/stream_popover.js:export function starred_messages_sidebar_menu_handle_keyboard(key) {
```

Reply:
Ahh, you'd deleted those lines in another comment -- anyway hopefully the fact that this is arrow key navigation within the popover is helpful. The feature is of low importance in this particular popover since there's no good way to open it from the keyboard, but several of our other popovers can be opened easily with a keyboard shortcut, and it's nice to just have keyboard navigation within all menu popovers work consistently, so I'd like to preserve that functionality.

Reply:
> it's nice to just have keyboard navigation within all menu popovers work consistently

I agree with you. I'll leave it as is.